docs/versions/1.0.0.md

```diff
@@ -148,32 +158,36 @@ Requirements
    in [ISO 8601]{:target=_blank} format, including the time zone and a millisecond precision time part. (c) The values
    in the `Created` timestamps SHOULD reflect the correct order of the versions.
 
-3. The `bag-info.txt` file MAY contain at most one element called `Is-Version-Of` with a
+3. The `bag-info.txt` file MAY contain exactly one element called `Data-Station-User-Account`. If provided, it MUST
+   contain the user name of the Data Station account that is authorized to deposit the bag.
```

Comment:
hmm, this authorization only makes sense in the context of a update-deposit. Or do you envision other restrictions?

Reply:
By the way, it seems this is not a stand-alone rule (i think that is why it was in Part 4)